vecgeom.spec

```diff
@@ -5,15 +5,20 @@ BuildRequires: cmake gmake
 
 %define isamd64 %(case %{cmsplatf} in (*amd64*) echo 1 ;; (*) echo 0 ;; esac)
 %define isaarch64 %(case %{cmsplatf} in (*_aarch64_*) echo 1 ;; (*) echo 0 ;; esac)
+%define isppc64 %(case %{cmsplatf} in (*_ppc64le_*) echo 1 ;; (*) echo 0 ;; esac)
```

Comment:
@mrodozov , can you please update the PR to use the following
`%ifarch ppc64le`

`%ifnarch ppc64le`

Reply:
Changes it everywhere and removed some defines